### PR TITLE
Bump CSV

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

File: CSV/versions/0.0.10/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4.1
+DataStreams 0.0.8
+DataFrames
```
Comment (tkelman) [inline CSV/versions/0.0.10/requires]: the package require file and the metadata requires content should match. this looks right, but the package copy removed things it shouldn't have

**Comment by quinnj** [comment]
What's the best way to fix this? Should I edit the CSV release tag to something like "broken release" and then do another v0.0.10 release? Is that ok to do?

**Comment by tkelman** [comment]
Since this hasn't gone into metadata, you can remove the 0.0.10 tag and re-do it if that's easiest. How did this mismatch happen?

**Comment by quinnj** [comment]
Ok, think I got this sorted out now.

**Pull request merged**